self-tests.yml

```diff
@@ -432,6 +437,15 @@ tasks:
             set -o errexit
 
             ./clients/linux_amd64/evergreen service deploy test-endpoints --commit ${revision} --username admin --key abb623665fdbf368a1db980dde6ee0f0
+  - name: js-tests
+    commands:
+      - command: git.get_project
+        type: system
+        params:
+          directory: gopath/src/github.com/evergreen-ci/evergreen
+      - func: run-make
+        vars:
+          target: "js-test"
```

Comment:
I would repalce this either with a new yaml anchor for run-js-test *or* just use the run go suite, which is basically, the same.

Reply:
done